### ISSUE TITLE
training data on hera and to linear baseline data

### ISSUE THREAD

**Issue by frolovsa** [frolovsa/model_error_correction#1]
TCC: Here's the original IFS replay and the reduced dataset: /scratch1/NCEPDEV/stmp2/Tse-chun.Chen/anal_inc/IFS_replay/. I think it would be good to move them up to the archive since it is taking up a lot of space and now that the paper is out there.

TCC: here is the npys directory: /scratch2/NCEPDEV/stmp1/Tse-chun.Chen/anal_inc/npys
This is where I store all the npys (but it is messy now...) The files you are looking for: ifs_f06_ranl_sub, ifs_sfc_ranl_sub, ifs_out_ranl_sub. They are the forecast fields, surface data, and analysis increments. The normalization uses the files that ends with *mean_1d.npy and *std_1d.npy

**Comment by frolovsa** [comment]
@tse-chunchen is there a separate directory with the linear baseline data?
